osbuild/__init__.py

```diff
@@ -1,490 +1,12 @@
 
-import contextlib
-import errno
-import hashlib
-import json
-import os
-import socket
-import shutil
-import subprocess
-import sys
-import tempfile
-import osbuild.remoteloop as remoteloop
+from .pipeline import Assembler, AssemblerFailed, load, Pipeline, Stage, StageFailed
 
 
 __all__ = [
     "Assembler",
     "AssemblerFailed",
-    "BuildRoot",
     "load",
     "Pipeline",
     "Stage",
     "StageFailed",
 ]
```

Comment:
Technically, we don't need `__all__` here, because the file is only importing a couple of names, which is easy to scan. But it seems to be [good form](https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces) to keep it anyway.